docs/RELEASE.txt

```diff
@@ -18,16 +18,22 @@ JDK
 This release was built and tested with OpenJDK 19. It uses modules and the
 newest jpackage for distribution.
 
+
 Major Enhancements
 ==============================================================================
-    * GH #121 Updated all icons with transpareancy
+    *
+
+    ***** Previous fixes *****
+    * GH #121 Updated all icons with transparancy
 
 Major Bug Fixes
 ==============================================================================
+    *
+
+    ***** Previous fixes *****
     * GH #119 (crashes opening HDF4 file) has been fixed. The problem was
       that the HDF4 xdr.dll file was not copied from the hdf4 install location.
 
-    ***** Previous fixes *****
     * HDFVIEW-284 (crashes on NETCDF-4 grids) has been fixed. The problem was
```

Comment:
I'm wondering if you think this sounds better:

"The problem was that HDFView was trying to handle references in variable-length containers like strings."?

Reply:
Or maybe this would be clearer: "The crashes were caused by HDFView handling references in variable-length containers  like strings.

Reply:
Maybe but it is old history anyway.